core/src/main/resources/error/error-classes.json

```
@@ -904,6 +904,11 @@
     ],
     "sqlState" : "42K05"
   },
+  "INVALID_ESC" : {
+    "message" : [
+      "Invalid escape string. Escape string must contain only one character."
```

Could you show the invalid string in the error message, please:
```suggestion
      "Found an invalid escape string: <invalidEscape>. The escape string must contain only one character."
```